src/cephadm/tox.ini

```diff
@@ -49,7 +49,7 @@ deps =
     flake8-quotes
 commands =
     flake8 --config=tox.ini {posargs:cephadm.py cephadmlib}
-    bash -c 'test $(git ls-files 'cephadm.py' 'cephadmlib/*.py' | sort -u | xargs grep "docker.io" | wc -l) == 11'
+    bash -c 'test $(git ls-files 'cephadm.py' 'cephadmlib/*.py' | sort -u | xargs grep "docker.io" | wc -l) == 12'
```

Comment:
🤯 btw, this won't work with tarballed sources (which ~is~ was the common when distributing sources).

Reply:
I wouldn't worry about this change. @rkachach had to update this line because of the new image. And this "test" is supposed to make sure we *know* how many docker.io based images are in the codebase (and discourage adding new ones ;-)). 

I also prefer writing tests like this using `ast` module or similar to avoid picking up on comments and whatnot... but this is what was there already. (That reminds me I want to at least file a PR that splits this off onto it's own environment, rather than piggybacking on flake8).

Reply:
Ok, something like this would be simpler:
```sh
bash -c 'test $(grep -r "docker.io" cephadm.py cephadmlib/*.py | wc -l) == 12'
```

Reply:
I think the `git ls-files` is used in order not to scrape "extra" stuff one may have lying around in the tree but not checked in. If you would like, let's take this discussion to a (new?) tracker or slack to discuss what to do wrt these lines longer term and not be a distraction to the mgmt-gateway discussion.

Reply:
@epuertat @phlogistonjohn yeah, this seems like a good side discussion to have. I'll open an issue so we can track it. In this PR I'm just updating the numbers to pass make check :)

Reply:
I opened https://tracker.ceph.com/issues/66686 to track this issue.